events/listener.go

```diff
@@ -35,3 +49,18 @@ func Listen(eventURL, accessKey, secretKey string, workerCount int) error {
 	err = router.StartWithoutCreate(nil)
 	return err
 }
+
+func checkTS(timestamps *time.Time) bool {
+	stampFile := config.Stamp()
+	if !utils.IsPathExist(stampFile) {
+		return true
+	}
+	// ignore the err cause the file exists
+	stats, _ := os.Stat(stampFile)
```

Comment:
If `err != nil` you should return true.  And then you don't need the IsPathExists above too.